packages/core/src/hooks/useAddressBook.tsx

```diff
@@ -85,25 +112,25 @@ export default function useAddressBook(): [
   return [addressBook, addContact, removeContact, getContactByContactId, editContact, getContactByAddress];
 }
 
-interface IAddressContact {
-  contactid: number;
+interface AddressContact {
```

Comment:
We prefer using `type` over `interface`. Should be a simple fix e.g.

```suggestion
type AddressContact = {
```

Reply:
Same applies for the other interfaces defined in this file